etc/org.opencastproject.videoeditor.impl.VideoEditorServiceImpl.cfg

```diff
@@ -12,7 +12,7 @@ video.codec = libx264
 # This is a temporary file, so it doesn't have to be generic, the default is ffmpeg only - lossless
 # ffmpeg.properties = -strict -2 -preset ultrafast -qp 0 -tune film
```

Comment:
Somewhat unrelated, but do you know what this line is about? Is it an example and if yes, what for?

Reply:
I think that's the somewhat badly formatted option `ffmpeg.properties`

Reply:
Right, so I was wondering why is present in line 13 (commented out), when it is also present in line 15 (commented in)

Reply:
Now I see what you mean. I'll change this line here as well:
```suggestion
# ffmpeg.properties = -preset ultrafast -qp 0 -tune film
```
And maybe then also do a follow-up pull request cleaning up this whole file.